tests/package.json

```diff
@@ -8,6 +8,7 @@
   "license": "BSD-4-Clause",
   "private": true,
   "dependencies": {
+    "busboy": "^0.3.1",
```

Comment:
Note that this will require that users re-run `npm install` in any existing installations of the test server.